gradoop-flink/src/main/java/org/gradoop/flink/io/impl/gdl/GDLDataSink.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.io.impl.gdl;
+
+import org.apache.flink.api.java.io.TextOutputFormat;
+import org.apache.flink.core.fs.FileSystem;
+import org.apache.flink.core.fs.Path;
+import org.gradoop.flink.io.api.DataSink;
+import org.gradoop.flink.io.impl.gdl.functions.GraphTransactionsToGDL;
+import org.gradoop.flink.model.api.epgm.GraphCollection;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+
+import java.io.IOException;
+
+/**
+ * A data sink that writes a graph or graph collection to a gdl formatted string.
+ * It is executed with a parallelism of 1, and therefore limited to smaller graphs.
+ * Shared elements in graph collections are converted to multiple elements in the resulting gdl.
```

Comment:
"Shared element in graph collections are printed multiple times but share a unique identifier. 
e.g. 
v0 with label "Person" contained in graph g0 and g1 
v1 with label "Person" contained only in graph g1 
Resulting GDLString: "
```
g0[
     v0_Person_0
]
g1[
     v0_Person_0
     v1_Person_1
]
```
"v0_Person_0 refers to the same materialized vertex contained in two graphs."